src/app/(payload)/admin/importMap.js

```diff
@@ -1,98 +1,65 @@
+import { TenantField as TenantField_1d0591e3cf4f332c83a86da13a0de59a } from '@payloadcms/plugin-multi-tenant/client'
```

Comment:
😞 Do we just always commit this?

Reply:
`pnpm prettify` will bring it down to the minimum set of changes - maybe you could actually just add a prettier run to the end of `pnpm generate:importmap` or `pnpm generate:types` since it's always going to be needed?

Reply:
We can do this in another PR. `prettier` removed the` importMap` file for now